grr/client/client_actions/linux/linux.py

```diff
@@ -169,7 +169,13 @@ class GetInstallDate(actions.ActionPlugin):
   out_rdfvalues = [rdf_protodict.DataBlob]
 
   def Run(self, unused_args):
-    self.SendReply(integer=int(os.stat("/lost+found").st_ctime))
+    ctime = 0
+    for path in ("/lost+found", "/"):
```

Comment:
there might be a better choice than / here, that's less likely to have had it's ctime changed after install. We should at least put a comment in about the edge conditions we know about (filesystems missing /lost+found).